src/ssl.c

```diff
@@ -5892,6 +5892,10 @@ int wolfSSL_CertManagerVerify(WOLFSSL_CERT_MANAGER* cm, const char* fname,
     if (file == XBADFILE) return WOLFSSL_BAD_FILE;
     if(XFSEEK(file, 0, XSEEK_END) != 0)
         return WOLFSSL_BAD_FILE;
+<<<<<<< HEAD
```

Comment:
Looks like merge conflict unresolved issues. Should be resolved before committing.